docs/pizza_insights.md

```diff
@@ -20,7 +20,7 @@ pizza insights <command> [flags]
 ### Options inherited from parent commands
 
 ```
-  -c, --config string       The saucectl config (default "~/.sauced.yaml")
+  -c, --config string       The codeowners config (default "~/.sauced.yaml")
```

Comment:
CODEOWNERS is a GitHub specific thing. Perhaps attributions configuration?

Reply:
I could do that. Every time I've seen it used specifically related to GH, it's CODEOWNERS in caps.

Reply:
@jpmcb thoughts?

Reply:
Hmmm "attributions" is probably more _correct_ but I wonder if "codeowners" will resonate more with people. I could go either way but "codeowners" feels abit better